rest-api/src/main/java/com/example/rest_api/controllers/FolderController.java

```diff
@@ -0,0 +1,63 @@
+package com.example.rest_api.controllers;
+
+import com.example.rest_api.entities.Bookmark;
+import com.example.rest_api.entities.Folder;
+import com.example.rest_api.repositories.FolderRepository;
+import lombok.extern.java.Log;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/folders")
+@Log
+public class FolderController {
```

Comment:
It's not in the scope of tasks 1&2, right? 😄

Reply:
yeah @mbnsky . I thought to keep up with weekly tasks so added it in the same pr. I've removed it now.